app/views/catalog/_index_default.html.erb

```diff
@@ -1,6 +1,6 @@
 <% doc_presenter = document_presenter(document) %>
 <%# default partial to display solr document fields in catalog index view -%>
-<abbr class="unapi-id" title="<%= document.id %>" aria-hidden="true"></abbr>
+<span class="Z3988" aria-hidden='true' title="<%= document.export_as_openurl_ctx_kev %>"></span>
```

Comment:
what is class Z3988 ? is it a random string?

Reply:
The Z3988 class is [part of the COinS standard](https://web.archive.org/web/20170424223448/http://ocoins.info/#id3205609419).  Apparently it is the NISO number for the OpenURL standard?

Reply:
It's part of the syntax of the COINS specification. https://en.wikipedia.org/wiki/COinS. I think it's an ISO code.